### PR TITLE
S3t4 fetch to update win

### PR DESCRIPTION
**Pull request by AlisonYoon**

### CODE REVIEW

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+    constructor(props) {
+        super(props)
+
+        this.state = {}
+    }
+
+    componentDidMount() {
+        this.setState({dog: ''})
+    }
+
```
[the remark this answers is not in the document]
Comment (AlisonYoon) [reply src/components/Card/Card.js]: but state can go

File: src/components/CardContainer/CardContainer.js [hunk 2]
```diff
@@ -36,10 +36,21 @@ class CardContainer extends React.Component {
...
+            method: 'POST'
+        })
+            .then(data => data.json())
+            .then(json => {
+                console.log(json)
+                console.log(id)
+            })
+    }
+
+    clickUpdateStateGetNewDogs = () => {
```
[the remark this answers is not in the document]
Comment (AlisonYoon) [reply src/components/CardContainer/CardContainer.js]: sure

File: src/components/CardContainer/CardContainer.js [hunk 3]
```diff
@@ -36,10 +36,21 @@ class CardContainer extends React.Component {
...
+                console.log(json)
+                console.log(id)
+            })
+    }
+
+    refreshDogs = () => {
         let dogs = [this.state.randomDog(), this.state.randomDog()]
         while ((dogs[0]._id == this.state.dogs[0]._id || dogs[0]._id == this.state.dogs[1]._id)
-            && (dogs[1]._id == this.state.dogs[0]._id || dogs[1]._id == this.state.dogs[1]._id)) {
+        && (dogs[1]._id == this.state.dogs[0]._id || dogs[1]._id == this.state.dogs[1]._id)) {
```
[the remark this answers is not in the document]
Comment (AlisonYoon) [reply src/components/CardContainer/CardContainer.js]: done

File: src/components/CardContainer/CardContainer.js [hunk 4]
```diff
@@ -36,10 +36,21 @@ class CardContainer extends React.Component {
...
-    clickUpdateWin = () => {
+    clickUpdateWin = (id) => {
+        fetch('http://localhost:3000/dogs/' + id + "/win", {
+            method: 'POST'
+        })
+            .then(data => data.json())
+            .then(json => {
+                console.log(json)
+                console.log(id)
+            })
```
Comment (MotNosniktaw) [inline src/components/CardContainer/CardContainer.js]: console.log(id) is unnecessary

Comment (mallowmew) [reply src/components/CardContainer/CardContainer.js]: neither console.log is necessary

File: src/components/CardContainer/CardContainer.js [hunk 5]
```diff
@@ -57,7 +68,10 @@ class CardContainer extends React.Component {
                             height={dog.height.metric + "cm"}
                             temperament={dog.temperament}
                             id={dog._id}
-                            clickEvent={this.clickUpdateWin}/>
+                            winCount={dog.winCount}
```
Comment (MotNosniktaw) [inline src/components/CardContainer/CardContainer.js]: winCount should be removed as all handled by api

**Pull request merged**